src/main/java/org/skriptlang/skript/lang/experiment/Experiment.java

```diff
@@ -0,0 +1,187 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.lang.experiment;
```

Comment:
We could likely avoid some of the NotNull annotations by including the default package info.

Reply:
But then we'd be using the same misused annotations and suppressions as we are elsewhere!!

Reply:
Having looked through, we'd be able to remove 4 `NotNull` annotations in this package, and we would then need 14 `UnknownNullability` annotations on everything else that is _not_ strictly NotNull and so I don't think this is a good change.